src/libs/ReportActionsUtils.js

```diff
@@ -228,6 +228,22 @@ function getLastClosedReportAction(reportActions) {
     return lodashFindLast(sortedReportActions, action => action.actionName === CONST.REPORT.ACTIONS.TYPE.CLOSED);
 }
 
+/**
+ * @param {Array} onyxData
+ * @returns {Object} The latest report action in the `onyxData` or `null` if one couldn't be found
+ */
+function getLatestReportActionFromOnyxData(onyxData) {
+    const reportActionUpdate = _.find(onyxData, onyxUpdate => onyxUpdate.key.startsWith(ONYXKEYS.COLLECTION.REPORT_ACTIONS));
+
+    if (!reportActionUpdate) {
+        return null;
```

Comment:
NAB: In general I think we prefer keeping the return type consistent by using an empty object instead, but it's not a big deal.